src/plugins/reporter-mantisbt.c

```diff
@@ -695,6 +695,7 @@ int main(int argc, char **argv)
                     if (g_list_find_custom(ii->mii_attachments, name, (GCompareFunc) strcmp) == NULL)
                         break;
 
+                    free(name);
```

Comment:
If you’re going to free it, then also unset the pointer. Something like `g_clear_pointer(&name, free)`.

Reply:
That assumes that there exists a path where the loop will reach the line on the last iteration. Not sure how plausible that is, but something to consider.

Reply:
Right, and there is no “last iteration”, because it just goes.